cmd/launcher/svc_windows.go

```diff
@@ -33,46 +31,28 @@ const serviceName = "launcher"
 
 // runWindowsSvc starts launcher as a windows service. This will
 // probably not behave correctly if you start it from the command line.
-func runWindowsSvc(args []string) error {
-	eventLogWriter, err := eventlog.NewWriter(serviceName)
-	if err != nil {
-		return fmt.Errorf("create eventlog writer: %w", err)
-	}
-	defer eventLogWriter.Close()
-
-	logger := eventlog.New(eventLogWriter)
-	logger = log.With(logger, "ts", log.DefaultTimestampUTC, "caller", log.DefaultCaller)
-
-	level.Debug(logger).Log(
-		"msg", "service start requested",
+func runWindowsSvc(systemSlogger *multislogger.MultiSlogger, args []string) error {
+	systemSlogger.Log(context.TODO(), slog.LevelInfo,
+		"service start requested",
 		"version", version.Version().Version,
 	)
 
 	opts, err := launcher.ParseOptions("", os.Args[2:])
 	if err != nil {
-		level.Info(logger).Log("msg", "Error parsing options", "err", err)
+		systemSlogger.Log(context.TODO(), slog.LevelInfo,
+			"error parsing options",
+			"err", err,
+		)
 		os.Exit(1)
 	}
 
-	// Now that we've parsed the options, let's set a filter on our eventLog logger.
-	// We don't want to set this on the teelogger below because we want debug logs to always
-	// go to debug.json.
-	if opts.Debug {
-		logger = level.NewFilter(logger, level.AllowDebug())
-	} else {
-		logger = level.NewFilter(logger, level.AllowInfo())
-	}
-
-	systemSlogger := multislogger.New(slog.NewJSONHandler(eventLogWriter, &slog.HandlerOptions{
-		Level: slog.LevelInfo,
-	}))
-
 	localSlogger := multislogger.New()
+	logger := log.NewNopLogger()
 
 	// Create a local logger. This logs to a known path, and aims to help diagnostics
 	if opts.RootDirectory != "" {
 		ll := locallogger.NewKitLogger(filepath.Join(opts.RootDirectory, "debug.json"))
-		logger = teelogger.New(logger, ll)
+		logger = ll
```

Comment:
Does anything use this `logger`, or is it just stale?

Reply:
It's mostly stale, but there are a couple places the logger is still in use. This change just amounts to only writing to debug.json when accessing `logger` instead of debug.json+event logs, so it should be a safe change.